RichCongressRecurrentFixturesTestBundle.php

```diff
@@ -16,4 +19,11 @@
 class RichCongressRecurrentFixturesTestBundle extends AbstractBundle
 {
     public const COMPILER_PASSES = [DataFixturesPass::class, TestAuthenticatorCompilerPass::class];
+
+    public function build(ContainerBuilder $container): void
```

Comment:
As you extends from the `AbstractCompilerPass`, please use the constant provided from this class instead of declaring explicitly. The constants can change the parameters used to add the CompilerPass. If it is still not enought, the method `AbstractCompilerPas::add` can be overriden